docs/Language Feature Status.md

```diff
@@ -0,0 +1,31 @@
+# Language Feature Status
+
+## C# 7.0 and VB 15
+
+| Feature | Branch | State | Owners |
+| ------- | ------ | ----- | ------ | ----- |
+| Async Main | none  | Feature Specification | [tyoverby](https://github.com/tyoverby), [agocke](https://github.com/agocke) |
+| Address of Static | none | Feature Specification | |
+| Binary Literals | [future](https://github.com/dotnet/roslyn/tree/future)  | Finishing |  |
+| Digit Separators | [future](https://github.com/dotnet/roslyn/tree/future)  | Finishing | |
+| [Local Functions](https://github.com/dotnet/roslyn/blob/future/docs/features/local-functions.md) | [future](https://github.com/dotnet/roslyn/tree/future)  | Finishing | [agocke](https://github.com/agocke), [jaredpar](https://github.com/jaredpar), [vsadov](https://github.com/vsadov) |
+| [Pattern Matching](https://github.com/dotnet/roslyn/blob/future/docs/features/patterns.md) | [features/patterns](https://github.com/dotnet/roslyn/tree/features/patterns) | Prototyping | [gafter](https://github.com/gafter), [alekseyts](https://github.com/alekseyts), [agocke](https://github.com/agocke) |
+| Records | none | Feature Specification | [jcouv](https://github.com/jcouv) |
+| Ref Returns | [future](https://github.com/dotnet/roslyn/tree/future) | Finishing | [vsadov](https://github.com/vsadov), [agocke](https://github.com/agocke), [jaredpar](https://github.com/jaredpar) |
+| Source Generation | [future](https://github.com/dotnet/roslyn/tree/features/generators) | Prototyping | [cston](https://github.com/cston), [vsadov](https://github.com/vsadov) |
+| Throw Expr | [features/patterns](https://github.com/dotnet/roslyn/tree/features/patterns) | Prototyping | [agocke](https://github.com/agocke), [tyoverby](https://github.com/tyoverby), [gafter](https://github.com/gafter) |
+| Tuples | [features/tuples](https://github.com/dotnet/roslyn/tree/features/tuples) | Prototyping | [vsadov](https://github.com/vsadov), [jcouv](https://github.com/jcouv) |
+| Out var | none | Feature Specification | [alekseyts](https://github.com/alekseyts) |
+| With Exprs | none | Feature Specification | [gafter](https://github.com/gafter) |
+
+## C# 7.0 and VB 15 + 1
```

Comment:
I'm pretty sure you don't mean to say that non-null ref types are planned for C# 7. Does the `and` operator in the section header have higher precedence than the `+`?